examples/guides/se-restful-webservice/README.adoc

```diff
@@ -250,208 +200,180 @@ import io.helidon.webserver.ServerRequest;
---
+// By default this will pick up application.yaml from the classpath
+Config config = Config.create(); //<1>
 
-=== Write your main class
-You need just a little more code so your app starts the Helidon web server
-and makes it aware of your greeting service.
+// Get webserver config from the "server" section of application.yaml
+ServerConfiguration serverConfig = //<2>
+        ServerConfiguration.create(config.get("server"));
 
-Add these Java `import` statements:
-[source,java]
-// _include::0-4:{main-class}[tags=importsStart;importsWebServer;importsEnd]
+WebServer server = WebServer.create(serverConfig, createRouting(config)); //<3>
 ----
-import io.helidon.config.Config;
-import io.helidon.webserver.Routing;
-import io.helidon.webserver.ServerConfiguration;
-import io.helidon.webserver.WebServer;
-import io.helidon.media.jsonp.server.JsonSupport;
-----
-. Add a field to your main class to hold a reference to a `GreetService` instance.
-+
+<1> Loads configuration from `application.yaml`.
+<2> Creates the `ServerConfiguration` object from the relevant part of the `Config`
+object just loaded.
+<3> Creates the server using the config and the updated routing rules (see below).
 --
-[source,java]
-// _include::0-0:{main-class}[tag=greetServiceDecl]
-----
-    private static GreetService greetService;
-----
---
-
-. Add a method to your main class to set up routing for your app.
+. Start the server.
 +
 --
 [source,java]
-// _include::0-6:{main-class}[tags=createRoutingStart;createRoutingBasic;registerGreetService;createRoutingEnd]
-----
-    private static Routing createRouting() {
-        greetService = new GreetService(); // <1>
-        return Routing.builder()
-                .register(JsonSupport.create()) // <2>
-                .register("/greet", greetService) // <3>
-                .build();
-    }
-----
-<1> Creates and saves the reference to the `GreetService`. (We'll use `greeting` reference again
-later when we add support for health checking.)
-<2> Tells the Helidon web server that you want to use JSON.
-<3> Associates the greeting service with the `/greet` path.
---
-
-. Add the `startServer` method.
+// _include::0-15:{main-class}[tag=startServer,indent=0]
+----
+// Try to start the server. If successful, print some info and arrange to
+// print a message at shutdown. If unsuccessful, print the exception.
+server.start() //<1>
+    .thenAccept(ws -> { //<2>
+        System.out.println(
+                "WEB server is up! http://localhost:" + ws.port() + "/greet");
+        ws.whenShutdown().thenRun(() //<3>
+            -> System.out.println("WEB server is DOWN. Good bye!"));
+        })
+    .exceptionally(t -> { //<4>
+        System.err.println("Startup failed: " + t.getMessage());
+        t.printStackTrace(System.err);
+        return null;
+    });
+
+// Server threads are not daemon. No need to block. Just react.
+----
+<1> Starts the server.
+<2> When the startup completes successfully print a message and...
```

Comment:
I suggest ending the sentence with "...print a message."

Reply:
I see your point about this (and your next comment). I was trying to reflect the fact that the code called out here 
1. waits for a successful start and then
   a. prints a "successful start" message, and
   b. arranges for a "shutdown" message to be printed if/when a normal shutdown occurs.

The code there does not, _at that moment_, print the shutdown message, but sets things up for it to print later.

Your proposed revision is briefer and I guess doesn't confuse things, but I wanted to explain my original more verbose version.